app/mailers/provider_mailer.rb

```diff
@@ -224,6 +224,22 @@ def find_service_is_now_open(provider_user)
     )
   end
 
+  def respond_to_applications_before_reject_by_default_date(provider_user)
+    @provider_user = provider_user
+    @reject_by_default_date = I18n.l(CycleTimetable.reject_by_default.to_date, format: :no_year)
+    @decline_by_default_date = I18n.l(CycleTimetable.decline_by_default_date.to_date, format: :no_year)
+    @notifications_path = Rails.application.routes.url_helpers.provider_interface_notifications_path
+    @applications_path = Rails.application.routes.url_helpers.provider_interface_applications_path
```

Comment:
I think this needs to be `Rails.application.routes.url_helpers.provider_interface_applications_url`

Reply:
And the application routes are included here to, I think.